js_modules/dagster-ui/packages/ui-core/src/assets/RecentUpdatesTimeline.tsx

```diff
@@ -39,12 +39,25 @@ type Props = {
 };
 
 export const RecentUpdatesTimelineForAssetKey = memo((props: {assetKey: AssetKey}) => {
-  const data = useRecentAssetEvents(
-    props.assetKey,
-    100,
-    MaterializationHistoryEventTypeSelector.ALL,
+  const data = useRecentAssetEvents(props.assetKey, 100, [
+    AssetEventHistoryEventTypeSelector.MATERIALIZATION,
+    AssetEventHistoryEventTypeSelector.FAILED_TO_MATERIALIZE,
+    AssetEventHistoryEventTypeSelector.OBSERVATION,
+  ]);
+  const materializations = data.events.filter(
+    (event) =>
+      event.__typename === 'MaterializationEvent' ||
+      event.__typename === 'FailedToMaterializeEvent',
+  );
+  const observations = data.events.filter((event) => event.__typename === 'ObservationEvent');
+  return (
+    <RecentUpdatesTimeline
+      assetKey={props.assetKey}
+      materializations={materializations}
+      observations={observations}
```

Comment:
If we separate these out it kind of defeats the purpose of putting them in a single list in GQL. Instead we need to make the component take the single list.

Reply:
agree. mainly wanted to make sure i could use this query as a drop in replacement without breaking a bunch of stuff before reworking the components to use the single list